extensions/smallrye-graphql/deployment/src/main/java/io/quarkus/smallrye/graphql/deployment/SmallRyeGraphQLConfig.java

```diff
@@ -19,6 +20,12 @@
     @ConfigItem(name = "metrics.enabled", defaultValue = "false")
     boolean metricsEnabled;
 
+    /**
+     * Change the type naming strategy. Default, MergeInnerClass, Full
```

Comment:
Is `TypeAutoNameStrategy` an enum? If so you can remove the default values, they will be automatically included in the doc.

Reply:
Cool. Thanks :)